src/qtt/algorithms/fitting.py

```diff
@@ -249,6 +250,25 @@ def gaussian_model(x, mean, sigma, amplitude):  # type: ignore
 
     return result_dict['fitted_parameters'], result_dict
 
+def fit_sine(x_data: np.ndarray, y_data: np.ndarray, initial_parameters = None) -> Tuple[Dict[str, Any],Dict[str, Any]]:
+    """ Fit a sine wave for the inputted data; see sine function in functions.py for model
```

Comment:
Add Args and Returns in docstring

Reply:
Also here two newlines before and after a method as pep dictates.